koku/masu/test/util/aws/test_common.py

```diff
@@ -412,6 +412,25 @@ def test_aws_post_processor(self):
         for column in TRINO_REQUIRED_COLUMNS:
             self.assertIn(column.replace("-", "_").replace("/", "_").replace(":", "_").lower(), columns)
 
+    def test_aws_post_processor_customer_filtered_columns(self):
+        """Test that customer filtered columns get converted correctly in the data frame."""
+        column_one = "bill_bill_type"
+        column_two = "line_item_usage_start_date"
+        expected_col_one = "bill_billtype"
+        expected_col_two = "lineitem_usagestartdate"
+        data = {column_one: [1, 2], column_two: [3, 4]}
+        data_frame = pd.DataFrame.from_dict(data)
+
+        processed_data_frame = utils.aws_post_processor(data_frame)
+        if isinstance(processed_data_frame, tuple):
+            processed_data_frame, df_tag_keys = processed_data_frame
+            self.assertIsInstance(df_tag_keys, set)
+
+        self.assertIn(expected_col_one, processed_data_frame)
+        self.assertIn(expected_col_two, processed_data_frame)
+        for column in TRINO_REQUIRED_COLUMNS:
+            self.assertIn(column.replace("-", "_").replace("/", "_").replace(":", "_").lower(), processed_data_frame)
```

Comment:
Another way to replace multiple strings without using `re` or chaining `replace` calls is to use a translation table.
```suggestion
        self.assertIn(expected_col_two, processed_data_frame)
        strings_to_replace = ("-", "/", ":")
        translation = {ord(s): ord("_") for s in strings_to_replace}
        for column in TRINO_REQUIRED_COLUMNS:
            self.assertIn(column.translate(translation).lower(), processed_data_frame)
```

Reply:
Not a blocker, just an FYI.